index.html

```diff
@@ -0,0 +1,221 @@
+								</span>
+							</div>
+							<!-- Content -->
+							<div class="col-sm-9">
+								<h3 class="h4"><a href="#" class="stretched-link no-undrln">Updates on Past Audits</a>
+								</h3>
+								<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+									incididunt ut labore et dolore magna.</p>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</section>
+
+<section class="pt-4 pb-4 bg-light">
+	<div class="container">
+		<div class="row pb-4">
+			<div class="col-sm-8 pr-4">
+				<h2>
+					Bringing togheter people, expertise, and technology for a better Canada,
+					<br> one audi at a time.
+				</h2>
+				<p class="lead">
+					The Office of the Auditor General of Canada (OAG) serves Parliament by providing it with objective,
+					fact-based
+					information and expert advice on government programs and activities, gathered through audits.
+					Parliamentarians
+					use OAG reports to oversee government activities and hold the federal government to account for its
+					handling of
+					public funds.
+				</p>
+			</div>
+			<div class="col-sm-4">
+				<img class="img-responsive" src="src/assets/entry-building.png" alt="Image of OAG building entry" />
+			</div>
+		</div>
+		<div class="row pb-4">
+			<div class="col-sm-5 pr-4">
+				<img class="img-responsive" src="src/assets/karen-hogan.png" alt="Photo of Karen Hogan" />
+			</div>
+			<div class="col-sm-7">
+				<blockquote>
+					<p class="lead">
+						<span class="text-danger">“</span>I have been and continue to be inspired by the people who work
+						in our office. They are caring,
+						creative,
+						skilled, incredibly intelligent and professional, and always focused on upholding high standards
+						while
+						contributing to a well-managed and accountable government. It has been my privilege to work
+						alongside
+						these
+						individuals and to have played a part in shaping the strategic direction of the office and
+						leading
+						organizational change.”
+						<br>
+						<span>— Karen Hogan</span><br> Auditor General of Canada
+					</p>
+				</blockquote>
+			</div>
+		</div>
+	</div>
+</section>
+<div class="bg-center bg-cover"
+	data-bgimg-srcset="https://wet-boew.github.io/vocab/wb/utilities#no-image 767w, src/assets/people-background2.png 768w">
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-6 pull-right pt-4 pb-4">
+				<h2>Working at the OAG</h2>
+				<p><strong>Challenging, meaningful work.</strong></p>
+				<p class="lead">
+					At the OAG, we always strive to be better. We aim to improve and adapt to the needs of our people—we
+					want you
+					to want to work here. We recognize the value of different educational, professional, and cultural
+					backgrounds
+					within our team.
+				</p>
+				<a class="btn btn-lg btn-primary" href="#">Career opportunities</a>
```

Comment:
Suggestion:

```suggestion
				<p><a class="stretched-link btn btn-lg btn-primary" href="#">Career opportunities</a></p>
```

Reply:
stretched-link it will not work in this case, since the clicable area will be wider than the buton link